src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -906,6 +907,25 @@ export default class ElementWrapper extends Wrapper {
 		});
 	}
 
+	add_styles(block: Block) {
+		this.node.styles.forEach(style_directive => {
+			
+			// @FIXME:
+			// I'm not sure how to set the right value expression here
+			// Hmm, I thought this would work:
+			
+			// const { name, expression: { node: expression } } = style_directive;
+			// const updater = b`@set_style(${this.var}, "${name}", ${expression}, false)`;
+	
+			
+			// @FIXME: This is obviously wrong, hardcoding the color
+			const { name } = style_directive;
+			const updater = b`@set_style(${this.var}, "${name}", "red", false)`;
+
+			block.chunks.hydrate.push(updater);
```

Comment:
also remember to check if the style directive expression has any dynamic dependencies, `style_directive.expression.dynamic_dependencies().length` and add to `block.chunks.update` if necessary.

Reply:
ah, good point, thank you!